### PR TITLE
Add IDR

### PR DESCRIPTION
**Pull request by simonbray**
A tool for accessing the Image Data Resource (repository of microscopy images) created by @wm75, @jburel and myself during a recent hackathon in Freiburg.

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/idr_download/idr_download_by_ids.xml [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+            <when value="--center">
+                <expand macro="region_spec" pos="center" />
+            </when>
+        </conditional>
+        <param name="skip_failed" type="boolean" checked="false" truevalue="--skip-failed" falsevalue=""
+        label="Skip failed retrievals?"
+        help="By default the tool will fail with an error on the first non-retrievable image ID. Here, you can choose to skip non-retrievable image IDs and continue downloading the available ones instead. The error log will contain warnings about failed IDs in this case." />
+    </inputs>
+    <outputs>
+        <data name="out_log" format="txt" label="${tool.name} error log" />
```
Comment (bgruening) [inline tools/idr_download/idr_download_by_ids.xml]: Does this needs to be an output by default?

Comment (wm75) [reply tools/idr_download/idr_download_by_ids.xml]: Well, it's the old collections issue. If the tool fails to download anything, then the empty collection has nothing to click on to diagnose the problem. The log can also contain lots of useful warnings, which would be harder to get at when they just end up in stderr.
All in all, I think the extra dataset does not hurt.

Comment (bgruening) [reply tools/idr_download/idr_download_by_ids.xml]: Ok!

**Comment by simonbray** [comment]
Thanks @bgruening!

**Pull request merged**